src/patternfly/components/Table/table-grid.scss

```diff
@@ -312,14 +312,14 @@
     }
 
     &.pf-m-expanded {
-      --pf-c-table--tbody--after--BorderLeftWidth: var(--pf-c-table--tbody--after--BorderWidth);
+      --pf-c-table--tbody--after--BorderLeftWidth: var(--pf-c-table--tbody--after--border-width--base);
 
       & tbody {
         --pf-c-table--tbody--after--BorderLeftWidth: 0;
       }
     }
 
-    > tr > :first-child:not(.pf-c-table__check)::before {
+    > tr > :first-child:not(.pf-c-table__check)::after {
       --pf-c-table__expandable-row--before--BorderLeftWidth: 0;
```

Comment:
i think this should be `--pf-c-table__expandable-row--after--BorderLeftWidth` because this var was removed